decidim-core/lib/decidim/core.rb

```diff
@@ -215,16 +218,6 @@ def self.seed!
     80
   end
 
-  # Exposes a configuration option: The maximum file size of an attachment.
-  config_accessor :maximum_attachment_size do
-    10.megabytes
-  end
-
-  # Exposes a configuration option: The maximum file size for user avatar images.
-  config_accessor :maximum_avatar_size do
-    5.megabytes
-  end
```

Comment:
@tramuntanal I've added the CHANGELOG note at 9422975c88e664951a7e4d8c15995e66406a8da7.

Reply:
And also note that there will be no startup crashes even if those configurations remain at the Decidim initializer. This is because you are modifying the `Decidim.config` hash in the startup which does not care if these settings are available or not.

It is also needed to keep these settings at the initializer if you want the migration to migrate those to the organization file upload settings in the database.